data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -271,6 +271,33 @@ t_result_with_preprint_version AS (
   FROM t_result_with_preprint_url_and_has_evaluations AS result
 ),
 
+# created to eliminate the same preprint version for different version of RP
+t_result_with_preprint_preprint_version AS (
+  SELECT
+    manuscript_id,
+    position_in_overall_stage,
+    long_manuscript_identifier,
+    preprint_version,
+    LAG(preprint_version) OVER (PARTITION BY manuscript_id ORDER BY position_in_overall_stage) AS previous_preprint_version
+  FROM t_result_with_initial_preprint_version
+  WHERE long_manuscript_identifier NOT LIKE '%-VOR-%'
+),
+
+t_result_with_preprint_version AS (
+  SELECT
+    *
+  FROM t_result_with_initial_preprint_version
+  WHERE position_in_overall_stage < COALESCE(
+    (
+      SELECT 
+        MIN(position_in_overall_stage)
```

Comment:
Maybe you could talk me through it or we could add a comment. I have to admit I find it a bit difficult to get my head around it.

An alternative could also be to do it in python (with unit tests).

Reply:
We can go through it in our meeting.